lib/Sema/CSDiagnostics.cpp

```diff
@@ -1477,3 +1477,18 @@ bool MissingMemberFailure::diagnoseAsError() {
   corrections.noteAllCandidates();
   return true;
 }
+
+bool AllowTypeOrInstanceMemberFailure::diagnoseAsError() {
```

Comment:
Since "fix" always leads to a solution we should be able to always suggest a fix-it to correct situation along with diagnostic.

Reply:
That will be handled by existing diagnostic code once I moved it over. Do you think we need new diagnostics for some cases?

Reply:
Last time I checked not all of them produced a fix-it unfortunately, you just need to be aware of that...